web/frontend/src/components/slam/2d-renderer.vue

```diff
@@ -1,47 +1,48 @@
 <script setup lang="ts">
 
 import { $ref } from 'vue/macros';
-import { threeInstance, MouseRaycaster, MeshDiscardMaterial } from 'trzy';
+import { threeInstance, MouseRaycaster, MeshDiscardMaterial, GridHelper } from 'trzy';
 import { onMounted, onUnmounted, watch } from 'vue';
 import * as THREE from 'three';
 import { MapControls } from 'three/examples/jsm/controls/MapControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
-import DestMarker from '../lib/destination-marker.txt?raw';
-import BaseMarker from '../lib/base-marker.txt?raw';
+import DestMarker from '@/lib/destination-marker.txt?raw';
+import BaseMarker from '@/lib/base-marker.txt?raw';
+import Legend from './2d-legend.vue';
 
+let points: THREE.Points | undefined;
 let pointsMaterial: THREE.PointsMaterial | undefined;
+let intersectionPlane: THREE.Mesh | undefined;
 
-const backgroundGridColor = 0xCA_CA_CA;
-
-const gridSubparts = ['AxesPos', 'AxesNeg', 'Grid'];
+const markerColor = '#FF0047';
+const backgroundGridColor = '#cacaca';
 
 const svgMarkerRenderOrder = 4;
 const pointsRenderOrder = 3;
 const axesHelperRenderOrder = 2;
 const gridHelperRenderOrder = 1;
 
-const cameraScale = 25;
+const cameraScale = 12.5;
```

Comment:
This was scaled back by two because we now do a multiplication against device pixel density, which on my computer was 2. On lower res monitors like Nick S.'s, it'll be 1.